tests/marqo_test.py

```diff
@@ -146,4 +142,39 @@ def warm_request(self, func, *args, **kwargs):
         This solves the occurence of tests failing due to eventual consistency implemented in marqo cloud.
         '''
         for i in range(5):
-            func(*args, **kwargs) 
+            func(*args, **kwargs)
+
+    def create_cloud_index(self, index_name, index_defaults=None, **kwargs):
+        client = marqo.Client(**self.client_settings)
+        index_settings = index_defaults if index_defaults else {}
+        index_settings.update({
+            "inference_type": "marqo.CPU", "storage_class": "marqo.basic", "model": "hf/all_datasets_v4_MiniLM-L6"
+        })
+        while True:
+            if client.http.get(f"/indexes/{index_name}/status") == "READY":
+                break
+            try:
+                client.create_index(index_name, settings_dict=index_settings)
```

Comment:
why do we try/break/except in a `while True` loop to create an index? 

Shouldn't `Client.create_index` method have logic for waiting on an index to be spun up?

Reply:
We are running try catch in case index already exists or deleting